massdriver/resource_package_alarm.go

```diff
@@ -34,6 +42,39 @@ func resourcePackageAlarm() *schema.Resource {
 				Type:        schema.TypeString,
 				Required:    true,
 			},
+			"metric": {
+				Type:     schema.TypeList,
+				MaxItems: 1,
+				Optional: true, // This should be removed when we've added it to all our existing alarms
+				//Required: false,     This should be set to true when we've added it to all our existing alarms
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": {
+							Type:        schema.TypeString,
+							Description: "Name of the metric. Required for all clouds.",
+							Required:    true,
+						},
+						"namespace": {
+							Type:        schema.TypeString,
+							Description: "Namespace of the metric. Required for AWS and Azure. Omit for GCP.",
```

Comment:
Required for GCP as well.

GCP metrics will look something like

```
      namespace: "cloudsql_database"
      name: "cloudsql.googleapis.com/database/cpu/utilization"
 ```
 
 namespace = monitored resource type
 name = metric type
 https://cloud.google.com/monitoring/mql/reference